.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Report a problem with this library that needs to be fixed.
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Before filing this issue**
+Make sure you've carefully reviewed the
+[documentation](https://developers.google.com/google-ads/api/docs/client-libs/python)
+for this library.
+
+This library is only used for the
+[Google Ads API](https://developers.google.com/google-ads/api/docs/start), for
+the [AdWords API](https://developers.google.com/adwords/api/docs/guides/start)
+client library report your issue
+[here](https://github.com/googleads/googleads-python-lib/issues).
+
+Only use this page to report problems with this library. If you are unsure
+whether your issue is related to this library or the Google Ads API itself,
+please first reach out to one of the below resources before filing an issue here:
+
+* [Post in the Support Forum](https://support.google.com/google-ads/thread/new)
+* [Email the Support Team](https://support.google.com/google-ads/gethelp)
```

Comment:
the email alias we use in blog posts is googleadsapi-support@google.com

Reply:
The page linked is probably better because it includes other remediation steps before sending an email, like offering relevant documentation, collecting the CID, etc.

Reply:
I disagree - that page is for generic Ads UI issues not API issues. That's why we use the email alias in blog posts.

Reply:
Fixed